river/ensemble/__init__.py

```diff
@@ -6,8 +6,11 @@
 aggregate predictions from different kinds of models.
 
 """
+<<<<<<< HEAD
```

Comment:
It seems you did a good job at merging the `main` branch. However, you're still missing a few cases. I would recommend search for all the `HEAD` mentions and fixing them manually :)

Reply:
Hi ! Yes, thank you ! I think it's fixed now.